lib/oxide-vpc/src/engine/overlay.rs

```diff
@@ -248,7 +246,16 @@ impl StaticAction for EncapAction {
         };
 
         let phys_target = match target {
-            RouterTargetInternal::InternetGateway => self.boundary_services,
+            RouterTargetInternal::InternetGateway => {
+                match self.v2b.get(&flow_id.dst_ip) {
+                    Some(phys) => PhysNet {
+                        ether: MacAddr::ZERO,
```

Comment:
For my own understanding/confirmation, this is because of the move to an anycast gateway IP?

Reply:
This has been reworked and this code no longer exists. Instead of doing anycast at the gateways, we're doing unicast and handling multipath decisions in OPTE based on flow hashing.